share/gpodder/ui/gtk/gpodderpreferences.ui

```diff
@@ -866,18 +898,32 @@
                                 <property name="visible">True</property>
                                 <property name="can-focus">True</property>
                                 <property name="receives-default">False</property>
+                                <property name="halign">start</property>
+                                <property name="margin-bottom">8</property>
```

Comment:
Should this be a top margin? There is a separator below it and the bottom margin creates a visual gap. It seems more logical to have a top margin that moves it away from the combo box above it.

Reply:
It is the last checkbox in the devices playlist section.

Reply:
At least on my system Gtk wraps the CheckButton label so that it expands only downwards (i.e. the label does not move with respect to the checkbox), and the space allocated to the CheckButton does not grow. If there is no free space below, then the label overlaps the separator when wrapped. This is probably a bug in Gtk.